temporalcli/commandsgen/commands.yml

```diff
@@ -3315,6 +3285,7 @@ option-sets:
           If unset, looks for an Update against the currently-running Workflow Execution.
 
   - name: update-targeting
+    description: Workflow update commands
```

Comment:
This seems wrong. Maybe "Commands that target Workflows for updates"? Not sure technically, but alignment with previous items, and not using the same description for two different things

Reply:
removing `description` [per comment above](https://github.com/temporalio/cli/pull/693/files#r1803621328)